src/Lumi/Components/List.purs

```diff
@@ -209,11 +209,6 @@ styles = jss
                 , justifyContent: "flex-end"
                 , alignItems: "center"
                 }
-            , "@media (max-width: 448px)":
-                { "& > lumi-list-row > lumi-list-row-cell":
-                    { alignItems: "flex-end"
-                    }
-                }
```

Comment:
Thinking about this a little more.. do we want to use `:last-child` or `:not(:first-child)` here instead of completely removing? We've got the right-align prop, but we might want the right or non-left columns to right-align even when the prop isn't set at small sizes.

Reply:
Hm... yeah that's a good point; I'll make that change